src/taoensso/timbre/appenders/3rd_party/newrelic.clj

```diff
@@ -0,0 +1,21 @@
+(ns taoensso.timbre.appenders.3rd-party.appenders.newrelic
+  "A New Relic appender"
+  {:author "Camilo Polymeris"}
+  (:import (com.newrelic.api.agent NewRelic)
+           (java.util HashMap)))
+
+(defn newrelic-appender
+  "New Relic appender. The Java agent must be installed and configured via newrelic.yml, see
+  https://docs.newrelic.com/docs/agents/java-agent/installation/java-agent-manual-installation#h2-install-agent"
+  [& _]
+  {:enabled?   true
+   :async?     true
+   :min-level  :warn
+   :rate-limit [[100 60000]]                                ;matches New Relic's own cap
+   :output-fn  :inherit
+   :fn         (fn [data]
+                 (let [{:keys [output-fn ?err level]} data
+                       params (HashMap. {"Log level" (name level)})]
+                   (if ?err
+                     (NewRelic/noticeError ?err params)
+                     (NewRelic/noticeError (output-fn data) params))))})
```

Comment:
> but I don't see how I could not want all errors to be recorded by New Relic.

Okay, so it actually makes sense to send *all* application errors from any source to New Relic? It's a little difficult from https://newrelic.com/ to figure out what this software actually does. Looked to me like some kind of metrics library but I must be misunderstanding?

What happens with the info that you report with `NewRelic/noticeError`?

To be absolutely clear: 

1. It makes sense to send _every_ application-level error to NewRelic even if the error has nothing to do with NewRelic itself?
2. It makes sense to send _only_ error-level information; there's no analog for other logging levels?

> Thank you for your patience.

No problem, and likewise thank you.